extensions-contrib/tdigestsketch/src/main/java/org/apache/druid/query/aggregation/tdigestsketch/sql/TDigestGenerateSketchSqlAggregator.java

```diff
@@ -145,14 +145,19 @@ private static class TDigestGenerateSketchSqlAggFunction extends SqlAggFunction
           null,
           OperandTypes.or(
               OperandTypes.ANY,
-              OperandTypes.and(
-                  OperandTypes.sequence(SIGNATURE_WITH_COMPRESSION, OperandTypes.ANY, OperandTypes.LITERAL),
-                  OperandTypes.family(SqlTypeFamily.ANY, SqlTypeFamily.NUMERIC)
-              )
+              // Validation for signatures like 'TDIGEST_GENERATE_SKETCH(column, compression)'
```

Comment:
There's no need to have the `OperandTypes.or` anymore. Just use the default checker with `requiredOperandCount(1)`.

Reply:
Resolved! Thanks for the review